shared/elements/gaia_switch/style.css

```diff
@@ -47,7 +47,7 @@ gaia-switch {
   content: '';
   position: absolute;
   right: 0;
-  top: 50%;
+  top: 2.5rem;
```

Comment:
I flash the device and see no obvious visual difference between previous and patched version, so I think the patch is good enough :)